classes/author/Author.inc.php

```diff
@@ -10,14 +10,14 @@
  * @class Author
  * @ingroup monograph
  *
- * @see AuthorDAO
+ * @see author\DAO
  *
  * @brief Monograph author metadata class.
  */
 
-namespace APP\monograph;
+namespace APP\author;
 
-use PKP\submission\PKPAuthor;
+use PKP\author\PKPAuthor;
```

Comment:
I didn't catch this before, but the base class should be `PKP\author\Author`. The `PKP` in `PKPAuthor` is not needed now that we use a separate namespace for it.

Reply:
Fixed